bin/appbundle-updater

```diff
@@ -154,7 +175,7 @@ CHEFDK_APPS = [
     "chef-dk",
```

Comment:
I think we can remove `chef-dk`

Reply:
removed in https://github.com/chef/appbundle-updater/pull/107/commits/6ee4d2613439292b58c64673a96a6302d90eac28.